modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/template/TemplateHandlerRegistryImpl.java

```diff
@@ -153,6 +155,7 @@ protected void deactivate() {
 
 	private final Map<TemplateHandler, ServiceRegistration<?>>
 		_serviceRegistrations = new ConcurrentHashMap<>();
+	private ServiceTrackerMap<String, ResourceBundleLoader> _serviceTrackerMap;
```

Comment:
Hey @Valmir-unicap, you do not need this tracker. Use the `ResourceBundleLoaderUtil` instead. Thx